[2025-04-09T15:26:06.067+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T15:26:06.084+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T15:25:43.596840+00:00 [queued]>
[2025-04-09T15:26:06.092+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T15:25:43.596840+00:00 [queued]>
[2025-04-09T15:26:06.093+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T15:26:06.179+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-04-09 15:25:43.596840+00:00
[2025-04-09T15:26:06.184+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=304) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T15:26:06.186+0000] {standard_task_runner.py:72} INFO - Started process 306 to run task
[2025-04-09T15:26:06.186+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'manual__2025-04-09T15:25:43.596840+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmppalrosyj']
[2025-04-09T15:26:06.187+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask run_dbt_trusted
[2025-04-09T15:26:06.268+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T15:25:43.596840+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T15:26:06.415+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T15:25:43.596840+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T15:25:43.596840+00:00'
[2025-04-09T15:26:06.416+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T15:26:06.429+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T15:26:06.432+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-04-09T15:26:06.440+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T15:26:06.442+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-09T15:26:06.443+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-04-09T15:26:08.684+0000] {subprocess.py:93} INFO - [0m15:26:08  Running with dbt=1.8.0
[2025-04-09T15:26:08.986+0000] {subprocess.py:93} INFO - [0m15:26:08  Registered adapter: postgres=1.8.2
[2025-04-09T15:26:09.543+0000] {subprocess.py:93} INFO - [0m15:26:09  Found 9 models, 10 sources, 428 macros
[2025-04-09T15:26:09.547+0000] {subprocess.py:93} INFO - [0m15:26:09
[2025-04-09T15:26:09.686+0000] {subprocess.py:93} INFO - [0m15:26:09  Concurrency: 1 threads (target='dev')
[2025-04-09T15:26:09.687+0000] {subprocess.py:93} INFO - [0m15:26:09
[2025-04-09T15:26:09.697+0000] {subprocess.py:93} INFO - [0m15:26:09  1 of 4 START sql table model driven_trusted.non_pii_data ....................... [RUN]
[2025-04-09T15:26:17.152+0000] {subprocess.py:93} INFO - [0m15:26:17  1 of 4 OK created sql table model driven_trusted.non_pii_data .................. [[32mSELECT 131012[0m in 7.45s]
[2025-04-09T15:26:17.155+0000] {subprocess.py:93} INFO - [0m15:26:17  2 of 4 START sql table model driven_trusted.payment_data ....................... [RUN]
[2025-04-09T15:26:18.830+0000] {subprocess.py:93} INFO - [0m15:26:18  2 of 4 OK created sql table model driven_trusted.payment_data .................. [[32mSELECT 131012[0m in 1.67s]
[2025-04-09T15:26:18.834+0000] {subprocess.py:93} INFO - [0m15:26:18  3 of 4 START sql table model driven_trusted.pii_data ........................... [RUN]
[2025-04-09T15:26:22.979+0000] {subprocess.py:93} INFO - [0m15:26:22  3 of 4 OK created sql table model driven_trusted.pii_data ...................... [[32mSELECT 131012[0m in 4.14s]
[2025-04-09T15:26:22.999+0000] {subprocess.py:93} INFO - [0m15:26:22  4 of 4 START sql table model driven_trusted.technical_data ..................... [RUN]
[2025-04-09T15:26:24.974+0000] {subprocess.py:93} INFO - [0m15:26:24  4 of 4 OK created sql table model driven_trusted.technical_data ................ [[32mSELECT 131012[0m in 1.99s]
[2025-04-09T15:26:24.994+0000] {subprocess.py:93} INFO - [0m15:26:24
[2025-04-09T15:26:24.995+0000] {subprocess.py:93} INFO - [0m15:26:24  Finished running 4 table models in 0 hours 0 minutes and 15.45 seconds (15.45s).
[2025-04-09T15:26:25.080+0000] {subprocess.py:93} INFO - [0m15:26:25
[2025-04-09T15:26:25.082+0000] {subprocess.py:93} INFO - [0m15:26:25  [32mCompleted successfully[0m
[2025-04-09T15:26:25.083+0000] {subprocess.py:93} INFO - [0m15:26:25
[2025-04-09T15:26:25.084+0000] {subprocess.py:93} INFO - [0m15:26:25  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-09T15:26:25.962+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T15:26:26.119+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T15:26:26.120+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=manual__2025-04-09T15:25:43.596840+00:00, execution_date=20250409T152543, start_date=20250409T152606, end_date=20250409T152626
[2025-04-09T15:26:26.196+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T15:26:26.212+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T15:26:26.323+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
